lib/node_modules/@stdlib/math/base/special/cceiln/test/test.js

```diff
@@ -47,111 +49,107 @@ tape( 'main export is a function', function test( t ) {
 
 	t.end();
 });
 
 tape( 'the function returns `+0` if provided `+0`', function test( t ) {
 	var v;
 
-	v = cceiln( +0.0, +0.0, 0 );
-	t.strictEqual( isPositiveZero( v[ 0 ] ), true, 'returns expected value' );
-	t.strictEqual( isPositiveZero( v[ 1 ] ), true, 'returns expected value' );
+	v = cceiln( new Complex128( +0.0, +0.0 ), 0 );
+	t.strictEqual( isPositiveZero( real( v ) ), true, 'returns expected value' );
+	t.strictEqual( isPositiveZero( imag( v ) ), true, 'returns expected value' );
 
-	v = cceiln( +0.0, +0.0, -2 );
-	t.strictEqual( isPositiveZero( v[ 0 ] ), true, 'returns expected value' );
-	t.strictEqual( isPositiveZero( v[ 1 ] ), true, 'returns expected value' );
+	v = cceiln( new Complex128( +0.0, +0.0 ), -2 );
+	t.strictEqual( isPositiveZero( real( v ) ), true, 'returns expected value' );
+	t.strictEqual( isPositiveZero( imag( v ) ), true, 'returns expected value' );
 
-	v = cceiln( +0.0, +0.0, 2 );
-	t.strictEqual( isPositiveZero( v[ 0 ] ), true, 'returns expected value' );
-	t.strictEqual( isPositiveZero( v[ 1 ] ), true, 'returns expected value' );
+	v = cceiln( new Complex128( +0.0, +0.0 ), 2 );
+	t.strictEqual( isPositiveZero( real( v ) ), true, 'returns expected value' );
+	t.strictEqual( isPositiveZero( imag( v ) ), true, 'returns expected value' );
 
 	t.end();
 });
 
 tape( 'the function returns `+infinity` if provided `+infinity`', function test( t ) {
-	var v = cceiln( PINF, PINF, 5 );
-	t.strictEqual( v[ 0 ], PINF, 'returns expected value' );
-	t.strictEqual( v[ 1 ], PINF, 'returns expected value' );
+	var v = cceiln( new Complex128( PINF, PINF ), 5 );
+	t.strictEqual( real( v ), PINF, 'returns expected value' );
+	t.strictEqual( imag( v ), PINF, 'returns expected value' );
 	t.end();
 });
 
 tape( 'the function returns `-infinity` if provided `-infinity`', function test( t ) {
-	var v = cceiln( NINF, NINF, -3 );
-	t.strictEqual( v[ 0 ], NINF, 'returns expected value' );
-	t.strictEqual( v[ 1 ], NINF, 'returns expected value' );
+	var v = cceiln( new Complex128( NINF, NINF ), -3 );
+	t.strictEqual( real( v ), NINF, 'returns expected value' );
+	t.strictEqual( imag( v ), NINF, 'returns expected value' );
 	t.end();
 });
 
 tape( 'the function supports rounding real and imaginary components to a desired number of decimals', function test( t ) {
-	t.deepEqual( cceiln( -PI, PI, -2 ), [ -3.14, 3.15 ], 'returns expected value' );
-	t.deepEqual( cceiln( PI, -PI, -2 ), [ 3.15, -3.14 ], 'returns expected value' );
-	t.deepEqual( cceiln( 9.99999, 9.9999, -2 ), [ 10.0, 10.0 ], 'returns expected value' );
-	t.deepEqual( cceiln( -9.99999, -9.99999, -2 ), [ -9.99, -9.99 ], 'returns expected value' );
-	t.deepEqual( cceiln( 0.0, 0.0, 2 ), [ 0.0, 0.0 ], 'returns expected value' );
-	t.deepEqual( cceiln( -12368.0, 12368.0, -3 ), [ -12368.0, 12368.0 ], 'returns expected value' );
+	t.deepEqual( cceiln( new Complex128( -PI, PI ), -2 ), new Complex128( -3.14, 3.15 ), 'returns expected value' );
```

Comment:
We need to compare the real and imag components separately. Deep equal won't work as you expect it here.